src/server/es/index.js

```diff
@@ -178,14 +182,18 @@ class ES {
             if (twoParts.length !== 2) return;
             const index = twoParts[0];
             const field = twoParts[1];
+            if (!this.fieldTypes || !this.fieldTypes[index] || !this.fieldTypes[index][field]) {
```

Comment:
Can you split the condition into separate, to have distinct error for no index and no field?